_content/canon/t0600.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "T0600 十善業道經: The Ten Wholesome Courses of Action"
+translator: "Upasaka Wong Mou-Lam"
+external_url: "https://ftp.budaedu.org/ebooks/pdf/CE011.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/1fB6KVwWegE3IjRD6bbDWNQ7Zfkq6jXyD/view?usp=drive_link"
+course: karma
+tags:
+  - bodhisatta
+  - indian
+publisher: "Buddha Educational Foundation"
+pages: 10
```

Comment:
Should have caught this in review (my bad!) but all works need a year, even if it's a guess.  Here I'll just go ahead and add 1940 as an educated guess (feel free to let me know if you have the exact year!)